net-voip/captagent/captagent-6.3.1.ebuild

```diff
@@ -0,0 +1,48 @@
+# Copyright 1999-2021 Gentoo Authors
+# Distributed under the terms of the GNU General Public License v2
+
+EAPI=7
+
+inherit autotools
+
+DESCRIPTION="CaptAgent:  100% Open-Source Packet Capture Agent for HEP"
+HOMEPAGE="https://sipcapture.org/"
+SRC_URI="https://github.com/sipcapture/captagent/archive/refs/tags/${PV}.tar.gz -> ${P}.tar.gz"
+
+LICENSE="GPL-3"
```

Comment:
Read the source files under upstreams src/ dir, GPL-3 doesn't seem exactly correct ;)

Reply:
Updated to GPL-3+

The Bison generated file doesn't specify a version, which means any version applies (as I understand).

There is also one file supplied by Aladdin, but it basically says for any purpose 'as-is' so it's not the standard a Aladdin license and I'm having trouble finding an associated one in ::gentoo/licenses/

Reply:
Updated to GPL-3+ which is more in line with most of the source files.  Two notable exceptions:

capplan.tab.c (generated from .y) which doesn't mention a version.
md5.c which basically says public domain as I interpret it.

I'm happy that we're allowed to redistribute it as part of a GPL-3+ project.  Please confirm.